From a8cada2024fb2c27ef48dd53a39f0c4b7cc63ee3 Mon Sep 17 00:00:00 2001
From: Anuj Mittal <anujx.mittal@intel.com>
Date: Thu, 29 Dec 2016 12:47:58 +0800
Subject: [PATCH] jhi: disable MEI driver and enable KDI

Upstream-Status: Inappropriate [Configuration]

Requested by Jonathan Lahav since MEI driver has
been replaced by KDI for APL-I.

MEI still remains the default driver for all other
use cases so this change can't be made upstream.

Signed-off-by: Anuj Mittal <anujx.mittal@intel.com>
---
 service/linux/jhi.conf | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/service/linux/jhi.conf b/service/linux/jhi.conf
index 8943837..c39b287 100644
--- a/service/linux/jhi.conf
+++ b/service/linux/jhi.conf
@@ -15,9 +15,9 @@
 #
 # Uncomment one of the following:
 #
-transport MEI
+#transport MEI
 #transport SOCKET
-#transport KERNEL
+transport KERNEL
 #
 #
 #
@@ -54,4 +54,4 @@ log_level RELEASE
 #
 #socket_path /var/run/jhi_socket
 socket_path /tmp/jhi_socket
-#
\ No newline at end of file
+#
-- 
1.9.1

